### PR TITLE
attempt to fix gameplay deletion errors

### PR DESCRIPTION
**Pull request by ceriddenn**
## **Description**

- hopefully fixes or if not gives us more insight to what might be the issue with gameplay deletion on prod

## **Issues**

-N/A

---

- [x] I have read the [Contributing Guide](https://docs.waldo.vision/en/getting-started/), and agree to follow the [Code of Conduct](https://docs.waldo.vision/legal/code-of-conduct/).

### CODE REVIEW

**Review by Huskydog9988 — APPROVED** [review]

**Review by Huskydog9988 — CHANGES_REQUESTED** [review]
migration needed??

File: packages/database/prisma/migrations/20230417221847_m4d17/migration.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-- DropForeignKey
+ALTER TABLE "Clip" DROP CONSTRAINT "Clip_gameplayId_fkey";
+
+-- AddForeignKey
+ALTER TABLE "Clip" ADD CONSTRAINT "Clip_gameplayId_fkey" FOREIGN KEY ("gameplayId") REFERENCES "Gameplay"("id") ON DELETE CASCADE ON UPDATE CASCADE;
```
Comment (Huskydog9988) [inline packages/database/prisma/migrations/20230417221847_m4d17/migration.sql:5]: this shouldn't cause data loss right?

**Review by Huskydog9988 — APPROVED** [review]

File: packages/database/prisma/migrations/20230417221847_m4d17/migration.sql (hunk 1 above)
Comment (ceriddenn) [reply packages/database/prisma/migrations/20230417221847_m4d17/migration.sql:5]: it shouldn't cause any data loss unless there a clip/segment documents, no?

Comment (Huskydog9988) [reply packages/database/prisma/migrations/20230417221847_m4d17/migration.sql:5]: i don't think so

Comment (ceriddenn) [reply packages/database/prisma/migrations/20230417221847_m4d17/migration.sql:5]: I'm double checking locally rn.

**Pull request merged**